auditlog/management/commands/auditlogmigratejson.py

```diff
@@ -0,0 +1,110 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.core.paginator import Paginator
+
+from auditlog.models import LogEntry
+
+
+class Command(BaseCommand):
+    help = "Migrates changes from changes_text to json changes."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-d",
+            "--database",
+            default=None,
+            help="If provided, we will use native db operations.",
+            dest="db",
+            type=str,
+            choices=["postgres", "mysql", "oracle"],
+        )
+        parser.add_argument(
+            "-b",
+            "--bactch-size",
+            default=None,
```

Comment:
isn't it better to have a default batch size?
I assume if the batch size is `None` it will process all the records in one shot am afraid it put pressure on databases with a lot of records.